packages/opds-browser/src/components/OPDSBrowser.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { createStore, applyMiddleware } from "redux";
+let thunk: any = require("redux-thunk");
+import reducers from "../reducers/index";
+import Root from "./Root";
+
+export default class OPDSBrowser extends React.Component<RootProps, any> {
+  store: Redux.Store;
+  root: any;
```

Comment:
maybe this is because the export is ConnectedRoot instead of Root... I find that confusing in general